src/senaite/databox/browser/view.py

```diff
@@ -171,7 +171,7 @@ def get_excel(self):
         """
         workbook = Workbook()
         first_sheet = workbook.get_active_sheet()
-        first_sheet.title = self.context.Title()
+        first_sheet.title = self.context.Title().decode('utf-8')
```

Comment:
Hi @toropok,
thanks for the fixture!
Would you mind to use the API function `api.safe_unicode(self.context.Title())` to convert to a unicode string?

Also don't forget to add a line into the Changelog:

- #29 Fix title encoding error when generating XLS worksheets

Reply:
thanks! and sorry didn't find changelog file again. :)